apps/api/src/app/events/usecases/send-message/digest/get-digest-events-regular.usecase.ts

```diff
@@ -10,12 +13,14 @@ export class GetDigestEventsRegular extends GetDigestEvents {
       _environmentId: command.environmentId,
       _id: command.jobId,
     });
+    if (!currentJob) throw new ApiException('Digest job is not found');
+
     const amount =
-      typeof currentJob?.digest.amount === 'number'
-        ? currentJob?.digest.amount
-        : parseInt(currentJob.digest.amount, 10);
+      typeof currentJob.digest?.amount === 'number'
+        ? currentJob.digest.amount
+        : parseInt(currentJob.digest?.amount ?? '0', 10);
     const earliest = sub(new Date(currentJob.createdAt), {
-      [currentJob.digest.unit]: amount,
+      [currentJob.digest?.unit ?? DigestUnitEnum.MINUTES]: amount,
```

Comment:
@p-fernandez so what's your suggestion? I can add the `// @ts-ignore` comments...
I'll revert it because it works with `NaN` and `undefined` preserving the date:
![Screenshot 2023-02-24 at 10 50 59](https://user-images.githubusercontent.com/2607232/221148005-a26ebfd5-1d76-4cb8-b0fb-74c498cc3aab.png)

Reply:
We should throw an exception to be honest. But it is not responsibility of this PR.
Please add the ignore comment and a TODO, please. 🙏🏻